website/react-magma-docs/src/pages/api/character-counter.mdx

```diff
@@ -40,15 +40,28 @@ import { Input } from 'react-magma-dom';
 
 export function Example() {
   return (
-    <Input
-      maxLength={4}
-      value="hi"
-      labelText="Inital Value Character Counter"
-    />
+    <Input maxCount={4} value="hi" labelText="Inital Value Character Counter" />
   );
 }
 ```
 
+## New Character Counter props
+
+Two new props have been added. The `hasCharacterCounter` prop which defaults to `true` and the `maxCount` prop which replaces `maxLength` and enables the Character Counter.
+
+In the interim if an input needs a native `maxlength` and not a Character Counter, set `hasCharacterCounter={false}` and then use `maxLength`.
+
+Please note that in the meantime, `maxLength` is still supported but will need to be changed to `maxCount` as future releases will remove `maxLength` and `hasCharacterCounter`.
```

Comment:
The changeset is the place to use this kind of voice, where we can explain things to developers in the form of what changed/what's been added (that's where they'll look for it). In our docs site, I think we need to be a bit more direct.

```suggestion
## maxLength vs maxCount

The `maxLength` prop temporarily controlled the Character Counter, but moving forward should exclusively be used for the HTML native behavior of preventing users of typing beyond the set maximum length.

To enable the Character Counter, use `maxCount` (preferred) or **both** `maxLength` and `hasCharacterCounter` (to be deprecated in future versions).
```